### ISSUE TITLE
Sort vulnerabilities by overdue SLA

### ISSUE THREAD

**Issue by dragid10** [DefectDojo/django-DefectDojo#6303]
**Is your feature request related to a problem? Please describe**
There is currently no way to sort vulnerabilities by overdue SLA. This functionality is useful when attempting to prioritize patching of vulnerabilities.

**Describe the solution you'd like**
As a user of DefectDojo, I want to be able to sort the open vulnerabilities by overdue SLA so that I can prioritize my patching strategy.

**Describe alternatives you've considered**
An alternative could be filtering specifically overdue vulnerabilities. That would likely require implementing a custom filter on the finding object

**Additional context**
I brought this up in [slack](https://owasp.slack.com/archives/C2P5BA8MN/p1651869465999809) a few days ago

**Comment by coheigea** [comment]
It would be good to include a new "SLA" option in the findings filter that would allow you to filter issues:

1. That are out of SLA
2. To input a number which would correspond to issues which will be out of SLA within the given number of days.

WDYT?

**Comment by dragid10** [comment]
> It would be good to include a new "SLA" option in the findings filter that would allow you to filter issues:
> 
> 1. That are out of SLA
> 2. To input a number which would correspond to issues which will be out of SLA within the given number of days.
> 
> WDYT?

Yeah I think both of those options are great! At a bare minimum, I think Feature 1 would be enough

**Comment by valentijnscholten** [comment]
The filter is already present, but sorting is only hapening client side.
